client/src/components/Navigation.js

```diff
@@ -0,0 +1,26 @@
+import { FaPaw } from "react-icons/fa";
+import { FaDog } from "react-icons/fa";
+import { FaMap } from "react-icons/fa";
+import { NavLink } from "react-router-dom";
+
+export default function Navigation() {
+  return (
+    <nav className="nav">
+      <button className="nav-button">
+        <NavLink exact to="/profile">
+          <FaPaw className="nav-icons" />
+        </NavLink>
+      </button>
+      <button className="nav-button">
+        <NavLink exact to="/friends">
+          <FaDog className="nav-icons" />
+        </NavLink>
+      </button>
+      <button className="nav-button">
```

Comment:
not toooo sure why you are putting the NavLink inside a button element. You can style it to look like a button without using the tag